di/getJRC_GEOPP_DB_csv.py

```diff
@@ -0,0 +1,137 @@
+        dict(name="FID", type="string")
+    )
+    # Must yield the modified datapackage
+    yield package.pkg
+    # And its resources
+    yield from package
+    
+def fillFID(row):
+    row["FID"] = "jrc_geopp_db-"+row["id_powerplant"]
+
+def add_extras_column(row):
+    extra_fields = {}
+    for key,value in row.items():
+        if key not in selected_fields:
+            extra_fields[key] = value
+    row["fields"] = json.dumps(extra_fields)
+
+
+def retrieve(*args):
+    return Flow(
+        # Load inputs
+        load(url, format='csv', ),
+        
+        # Process them (if necessary)
+        update_package(name="jrc_geopp_db_csv"),
+        update_resource('JRC-GEOPP-DB', name='jrc_geopp_db', path='jrc_geopp_db.csv'),
+        add_extras_to_schema,
+        add_extras_column,
+        addFID,
+        fillFID,
+        # printer(num_rows=10),
+        select_fields(selected_fields, resources=None),
+        # printer(),
+        # add_computed_field(target=dict(name='geometry', type='geopoint'), operation='format', with_='{longitude}, {latitude}'),
+        add_computed_field(target=dict(name='geometry', type='string'), operation='format', with_='POINT({longitude} {latitude})'),
+        delete_fields(['latitude', 'longitude']),
+        
+        # # Update metadata
+        set_primary_key('id_powerplant'),
+        update_package(sources = [{
+                                    "title": "Joint Research Centre Data Catalogue",
+                                    "path": "https://data.jrc.ec.europa.eu/dataset/jrc-10128-10001"
+                                }]),
+        # printer(),
+        # Resource 1
+        duplicate(source="jrc_geopp_db", target_name="jrc_geopp_db_spatial", target_path="jrc_geopp_db_spatial.csv"),
+        delete_fields(selected_fields_spatial,resources=1),
+        # printer(num_rows=10),
+
+        # Resource 0
+        unpivot(unpivoting_fields, extra_keys, extra_value, regex=True,  resources=0),
+        delete_fields(['geometry'],resources=0),
+        find_replace([{"name": "value", "patterns":[{"find":"NULL","replace":""}]}], resources=0),
+        # update_schema(None, missingValues=["NULL", ""], resources=0),
+        set_type('value',type="number", resources=0),
+        # validate()
+        # printer(num_rows=10),
+    )
+
+
+
+if __name__ == '__main__':
+    host = "enermaps_db_1"
+    port = 5432
+    ds_id = 2
+    # host = "localhost"
+    # port = 5433
+    Flow(retrieve(),
+        dump_to_path(out_path='data/jrc_geopp_db',format="csv")
+        # dump_to_sql({"spatial":{"resource-name":"jrc_geopp_db_spatial", "mode":"append"},
+        #             "data":{"resource-name":"jrc_geopp_db", "mode":"append"}},
+        #             engine="postgresql://test:example@{host}:{port}/dataset".format(host=host,port=port))
+        ).process()
+    
+    import pandas as pd
+    import geopandas as gpd
+    import utilities
+    from shapely import wkt
+    import shutil
```

Comment:
I don't understand the reasoning behind putting those import here and not at the top of the file.